### PR TITLE
Adjusts left and right flywheel speeds to be different

### PR DESCRIPTION
**Pull request by redPlover**
This will supposedly lead to greater shooting accuracy.

### CODE REVIEW

**Comment by jkleiber** [comment]
Is there anything we need to adjust in SimIO as well (just so we can see that this works)

**Comment by jkleiber** [comment]
@redPlover I think we should add the offset in ShooterIOSim as well just so we can verify in advantage scope that the shooter wheel velocities are different (and for the sake of consistency between sim and reality)

**Comment by redPlover** [comment]
> @redPlover I think we should add the offset in ShooterIOSim as well just so we can verify in advantage scope that the shooter wheel velocities are different (and for the sake of consistency between sim and reality)

@jkleiber Okay, just refactored them to be controlled separately

File: src/main/java/frc/robot/subsystems/scoring/ShooterIOSim.java [hunk 1]
```diff
@@ -41,21 +54,36 @@ public void setKickerVolts(double volts) {
...
     public void updateInputs(ShooterIOInputs inputs) {
-        shooterSim.update(Constants.loopTime);
+        shooterLeftSim.update(Constants.loopTime);
+        shooterRightSim.update(Constants.loopTime);
+
+        shooterLeftAppliedVolts =
+                shooterFeedforward.calculate(shooterLeftSim.getAngularVelocityRadPerSec())
+                        + shooterLeftController.calculate(
+                                shooterLeftSim.getAngularVelocityRadPerSec(),
+                                shooterLeftGoalVelRPM * 2.0 * Math.PI / 60.0);
```
Comment (jkleiber) [inline src/main/java/frc/robot/subsystems/scoring/ShooterIOSim.java:64]: Could we add a constant for converting RPM to rad/s (similar to `kRadiansPerSecondtoRPM`)?

(would also apply below)

File: src/main/java/frc/robot/subsystems/scoring/ShooterIOTalon.java [hunk 2]
```diff
@@ -60,12 +63,18 @@ public void setKickerVolts(double volts) {
...
-        shooterLeft.setControl(controller.withVelocity(goalVelocityRPM));
-
-        inputs.shooterVelocityRPM = shooterLeft.getVelocity().getValueAsDouble() * 60;
-        inputs.shooterGoalVelocityRPM = goalVelocityRPM;
-        inputs.shooterAppliedVolts = shooterLeft.getMotorVoltage().getValueAsDouble();
-        inputs.shooterCurrentAmps = shooterLeft.getSupplyCurrent().getValueAsDouble();
+        shooterLeft.setControl(leftController.withVelocity(goalLeftVelocityRPM));
+        shooterRight.setControl(rightController.withVelocity(goalRightVelocityRPM));
+
+        inputs.shooterLeftVelocityRPM = shooterLeft.getVelocity().getValueAsDouble() * 60;
```
Comment (jkleiber) [inline src/main/java/frc/robot/subsystems/scoring/ShooterIOTalon.java:69]: see other comment about unit conversion constant

**Review by jkleiber — COMMENTED** [review]
First pass review

**Review by jkleiber — APPROVED** [review]

**Pull request merged**